<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaJuego</class>
 <widget class="QWidget" name="VentanaJuego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DCCurve</string>
  </property>
  <widget class="QFrame" name="grupo_elegir_teclas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>591</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="text_titulo_jugador">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jugador</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="text_indicaciones_1">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>20</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Presione el botón correspondiente al tipo de giro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_izquierda_jugador">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_indicaciones_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>50</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Luego, presione la tecla con la cuál desea realizar el movimiento</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_derecha_jugador">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_derecha_invitado">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_titulo_invitado">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Invitado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_listo_teclas">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Listo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_izquierda_invitado">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <zorder>game_zone</zorder>
   <zorder>game_zone</zorder>
   <zorder>game_zone</zorder>
   <zorder>game_zone</zorder>
   <zorder>text_titulo_jugador</zorder>
   <zorder>text_indicaciones_1</zorder>
   <zorder>button_izquierda_jugador</zorder>
   <zorder>text_indicaciones_2</zorder>
   <zorder>button_derecha_jugador</zorder>
   <zorder>button_derecha_invitado</zorder>
   <zorder>text_titulo_invitado</zorder>
   <zorder>button_listo_teclas</zorder>
   <zorder>button_izquierda_invitado</zorder>
   <zorder>espacio_cargar</zorder>
   <zorder>grupo_juego</zorder>
   <zorder>fondo_elegir_teclas</zorder>
  </widget>
  <widget class="QLabel" name="fondo_elegir_teclas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="text_esperando">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Esperando a que el resto esté listo :D</string>
   </property>
  </widget>
  <widget class="QFrame" name="grupo_juego">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="game_zone">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>8</y>
      <width>461</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_nueva_partida">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver al Menú de Partidas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pausa">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Pausar</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_puntaje_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_puntaje_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_jugador_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_jugador_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_jugador_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_jugador_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_puntaje_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_puntaje_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_salir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_puntaje_max">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Puntaje Máxmo: </string>
    </property>
   </widget>
   <zorder>grupo_elegir_teclas</zorder>
   <zorder>grupo_elegir_teclas</zorder>
   <zorder>grupo_elegir_teclas</zorder>
   <zorder>game_zone</zorder>
   <zorder>button_nueva_partida</zorder>
   <zorder>button_pausa</zorder>
   <zorder>text_puntaje_3</zorder>
   <zorder>text_puntaje_4</zorder>
   <zorder>text_jugador_1</zorder>
   <zorder>text_jugador_3</zorder>
   <zorder>text_jugador_2</zorder>
   <zorder>text_jugador_4</zorder>
   <zorder>text_puntaje_2</zorder>
   <zorder>text_puntaje_1</zorder>
   <zorder>button_salir</zorder>
   <zorder>text_puntaje_max</zorder>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>320</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>320</y>
     <width>6</width>
     <height>6</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_poder_1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_poder_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_poder_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_poder_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>12</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>grupo_juego</zorder>
  <zorder>fondo_elegir_teclas</zorder>
  <zorder>grupo_elegir_teclas</zorder>
  <zorder>text_indicaciones_1</zorder>
  <zorder>text_esperando</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_poder_1</zorder>
  <zorder>label_poder_2</zorder>
  <zorder>label_poder_3</zorder>
  <zorder>label_poder_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
